packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -217,14 +206,14 @@ declare global {
        * @param document
        * @param resolver
        */
-      stubMutationResolver<Result, Variables, T extends TypedDocumentNode<Result, Variables>>(
+      stubMutationResolver<T extends TypedDocumentNode<any, any>>(
         document: T,
         resolver: MutationResolver<T>
       ): Cypress.Chainable<ClientTestContext>
       /**
        * Mount helper for a component with a GraphQL fragment, as a list
        */
-      mountFragmentList<Result, Variables, T extends TypedDocumentNode<Result, Variables>>(
+      mountFragmentList<T extends TypedDocumentNode<any, any>>(
```

Comment:
at first i thought this weakened types, but i guess this doesn't weaken types because `Result` and `Variables` are already `any` (and unused)? is that correct?

Reply:
Right, yeah `T` extends that but the concrete values are still kept, there's new helpers that we use to extract the types from that (`ResultOf, VariablesOf`). The generics removed here previously was just another way to do the same but isn't quite compatible w/ the new helper approach.